lib/Logger/LoggerFeature.cpp

```diff
@@ -199,10 +199,15 @@ void LoggerFeature::collectOptions(std::shared_ptr<ProgramOptions> options) {
       new BooleanParameter(&_shortenFilenames),
       arangodb::options::makeDefaultFlags(arangodb::options::Flags::Hidden));
   
+  options->addOption("--log.hostname",
+                     "hostname to use in log message (empty for none, use 'auto' to automatically figure out hostname)",
+                     new StringParameter(&_hostname))
+                     .setIntroducedIn(30800);
+  
   options->addOption("--log.process", "show process identifier (pid) in log message",
                      new BooleanParameter(&_processId),
                      arangodb::options::makeDefaultFlags(arangodb::options::Flags::Hidden))
-                     .setIntroducedIn(30701);
+                     .setIntroducedIn(30800);
```

Comment:
Is this related/intended?

Reply:
Yes, this is intended. I found this while adding the new option, and found that the already existing option `--log.process` was only added in 3.8, but not in 3.7.1.
So I fixed it.